package.json

```diff
@@ -40,9 +40,11 @@
     "prettier": "1.11.1",
     "shelljs": "^0.8.1",
     "shx": "^0.2.2",
+    "systemjs": "^0.21.0",
+    "systemjs-plugin-babel": "0.0.25",
     "web-ext": "^2.4.0",
     "webpack": "^4.1.1",
     "webpack-cli": "^2.0.10"
   },
-  "license": "MPL-2.0"
+  "license": "MPL-2.0",
```

Comment:
Ah, I guess a trailing comma makes the JSON parser sad.

Reply:
Yep. Apparently I went spastic before pushing ... Now I'm having all kinds of merge issues before it'll let me push a fix. I'll address the other stuff, too, while I'm at it!